packages/services/api/src/modules/audit-logs/providers/audit-logs-types.ts

```diff
@@ -332,8 +332,20 @@ export const AuditLogModel = z.union([
     eventType: z.literal('ORGANIZATION_ACCESS_TOKEN_CREATED'),
     metadata: z.object({
       organizationAccessTokenId: z.string().uuid(),
-      permissions: z.array(z.string()),
+      permissions: z.array(z.string()).nullable(),
       assignedResources: ResourceAssignmentModel,
+      userId: z
+        .string()
+        // optional, because it was introduced later on.
+        .optional()
+        .nullable()
+        .transform(value => value ?? null),
+      projectId: z
```

Comment:
(not really on this line)

I wanted to confirm that we have audit log event for deletion of tokens, because now an owner/admin account could delete tokens that are created by other users, and this needs to be logged in the audit log.

Reply:
its on the follow up line :D